lightning/src/ln/channelmanager.rs

```diff
@@ -5703,6 +5703,25 @@ impl<M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelManager<M, T, K, F
 		}
 	}
 
+	/// Gets inflight HTLC information by processing pending `HTLCSource`s that are in
+	/// `Channel::pending_outbound_htlcs`
```

Comment:
```suggestion
	/// Gets inflight HTLC information by processing pending outbound `HTLCSource`s that are in
	/// our channels
```
Since they also come from the holding cell